lib/array/uniq.js

```diff
@@ -37,7 +38,8 @@
      */
     module.exports = function uniq(array, iteratee, thisArg) {
         assertArgument(array = toArray(array), 1, 'Arrayable');
-        assertArgument(isFunction(iteratee) || isObject(iteratee) || isString(iteratee), 2, 'Function, Object or string');
+        assertArgument(isFunction(iteratee) || isObject(iteratee) || isString(iteratee) || isVoid(iteratee), 2, 'Function, Object or string');
```

Comment:
isVoid should be first as it will make an early exit when no iteratee is passed (most of the time).